src/warnet/control.py

```diff
@@ -184,72 +206,45 @@ def run(scenario_file: str, additional_args: tuple[str]):
-            "metadata": {
-                "name": "warnetjson",
-                "namespace": namespace,
-            },
-            "data": {"warnet.json": json.dumps(tanks)},
-        },
-        {
-            "apiVersion": "v1",
-            "kind": "ConfigMap",
-            "metadata": {
-                "name": "scenariopy",
-                "namespace": namespace,
-            },
-            "data": {"scenario.py": scenario_text},
-        },
-        {
-            "apiVersion": "v1",
-            "kind": "Pod",
-            "metadata": {
-                "name": name,
-                "namespace": namespace,
-                "labels": {"mission": "commander"},
-            },
-            "spec": {
-                "restartPolicy": "Never",
-                "containers": [
-                    {
-                        "name": name,
-                        "image": "bitcoindevproject/warnet-commander:latest",
-                        "args": additional_args,
-                        "volumeMounts": [
-                            {
-                                "name": "warnetjson",
-                                "mountPath": "warnet.json",
-                                "subPath": "warnet.json",
-                            },
-                            {
-                                "name": "scenariopy",
-                                "mountPath": "scenario.py",
-                                "subPath": "scenario.py",
-                            },
-                        ],
-                    }
-                ],
-                "volumes": [
-                    {"name": "warnetjson", "configMap": {"name": "warnetjson"}},
-                    {"name": "scenariopy", "configMap": {"name": "scenariopy"}},
-                ],
-            },
-        },
-    ]
-    with tempfile.NamedTemporaryFile(mode="w", suffix=".yaml", delete=False) as temp_file:
-        yaml.dump_all(kubernetes_objects, temp_file)
-        temp_file_path = temp_file.name
 
-    if apply_kubernetes_yaml(temp_file_path):
-        print(f"Successfully started scenario: {scenario_name}")
-        print(f"Commander pod name: {name}")
-    else:
-        print(f"Failed to start scenario: {scenario_name}")
+    # Encode warnet data
+    warnet_data = base64.b64encode(json.dumps(tanks).encode()).decode()
 
-    os.unlink(temp_file_path)
+    try:
+        # Construct Helm command
+        helm_command = [
+            "helm",
+            "upgrade",
+            "--install",
+            "--namespace",
+            namespace,
+            "--set",
+            f"fullnameOverride={name}",
+            "--set",
+            f"scenario={scenario_data}",
+            "--set",
+            f"warnet={warnet_data}",
+        ]
+
+        # Add additional arguments
+        if additional_args:
+            helm_command.extend(["--set", f"args={' '.join(additional_args)}"])
+
+        helm_command.extend([name, "resources/charts/commander"])
```

Comment:
this fails when i try to run a scenario from a project directory (not in warnet code repo)

```
(.venv) --> warnet run scenarios/miner_std.py --mature --allnodes --interval 1
Failed to start scenario: miner_std
Error: Error: repo resources not found
```

Reply:
Thanks nice spot. Need to use `files()`